### ISSUE TITLE
Picking with Globe Materials / Globe Lighting enabled makes the shader compilation fail

### ISSUE THREAD

**Issue by shehzan10** [AnalyticalGraphicsInc/cesium#6039]
This can be reproduced by using the globe materials sandcastle and just clicking anywhere on the terrain.

The shader compilation failure log is:
```
[Cesium WebGL] Vertex shader compile log: ERROR: 0:177: 'normalMC' : undeclared identifier
ERROR: 0:177: '=' : dimension mismatch
ERROR: 0:177: '=' : cannot convert from 'const highp float' to 'highp 3-component vector of float'

[Cesium WebGL] Vertex shader translation failed.
Uncaught RuntimeError: Vertex shader failed to compile.  Compile log: ERROR: 0:177: 'normalMC' : undeclared identifier
ERROR: 0:177: '=' : dimension mismatch
ERROR: 0:177: '=' : cannot convert from 'const highp float' to 'highp 3-component vector of float'

Error
    at new RuntimeError (http://localhost:8004/Source/Core/RuntimeError.js:42:19)
    at createAndLinkProgram (http://localhost:8004/Source/Renderer/ShaderProgram.js:246:23)
    at initialize (http://localhost:8004/Source/Renderer/ShaderProgram.js:451:23)
    at ShaderProgram._bind (http://localhost:8004/Source/Renderer/ShaderProgram.js:468:9)
    at beginDraw (http://localhost:8004/Source/Renderer/Context.js:937:12)
    at Context.draw (http://localhost:8004/Source/Renderer/Context.js:1000:9)
    at DrawCommand.execute (http://localhost:8004/Source/Renderer/DrawCommand.js:502:17)
    at executeCommand (http://localhost:8004/Source/Scene/Scene.js:1722:21)
    at executeCommands (http://localhost:8004/Source/Scene/Scene.js:1977:17)
    at executeCommandsInViewport (http://localhost:8004/Source/Scene/Scene.js:2472:9) (on line 246 of http://localhost:8004/Source/Renderer/ShaderProgram.js)
```

Using Cesium version 1.40.

**Comment by hpinkos** [comment]
Good catch @shehzan10, thanks!  I marked this issue as `priority - next release` 

@jasonbeverage @bagnell any idea what might be happening here?

**Comment by lilleyse** [comment]
I think it is related to this comment: https://github.com/AnalyticalGraphicsInc/cesium/pull/5919#discussion_r148866827

My guess is the globe material code, which requires normals, is being added to the pick shader, which doesn't include normals. The fix would be to not apply materials to the pick shader.

This is essentially the same issue as the linked comment. When `enableLighting` is false the regular shader doesn't include normals, but the globe material expects them to exist.

I should be able to get to this before the next release.

**Comment by hpinkos** [comment]
Great, thanks @lilleyse!

**Comment by pjcozzi** [comment]
@lilleyse - as long as the material doesn't `discard`...

**Comment by pjcozzi** [comment]
Thanks again for the report, @shehzan10.  This is fixed in #6063.
